lib/wanda/execution/execution_result_schema.ex

```diff
@@ -9,9 +9,9 @@ defmodule Wanda.Results.ExecutionResult do
 
   @primary_key false
   schema "execution_results" do
-    field :execution_id, Ecto.UUID, primary_key: true
-    field :group_id, Ecto.UUID
-    field :payload, :map
+    field(:execution_id, Ecto.UUID, primary_key: true)
```

Comment:
something went wrong with the formatter, parenthesis should not appear here, add the ecto format dep to .formatter.exs

Reply:
you mean this?

https://github.com/trento-project/wanda/blob/main/.formatter.exs#L3
```exs
# Used by "mix format"
[
  import_deps: [:ecto, :phoenix],
  inputs: ["*.{ex,exs}", "priv/*/seeds.exs", "{config,lib,test}/**/*.{ex,exs}"],
  subdirectories: ["priv/*/migrations"]
]
```
What am I missing?

Reply:
maybe the formatter did something bad, try to remove parenthesis and format it again

Reply:
formatter went bananas :smile: